src/com/itacademy/eshop/simulations/ManagerSimulation.java

```diff
@@ -27,25 +27,30 @@ private void addThreeNewProducts() {
         /**
          * adds three new products to the shop. One of them should be a book, one should be a Laptop, and one should be a shirt.
          */
-        Product laptop = new Product("Laptop", 1000, Category.ELECTRONICS);
+        shop.addProduct(new Product("Laptop", 1000, Category.ELECTRONICS));
+        shop.addProduct(new Product("Bible", 10, Category.BOOKS));
+        shop.addProduct(new Product("Shirt", 15, Category.CLOTHING));
     }
 
     private void findAndRemoveOneProduct() {
         /**
          * manager searcher for a product with the name "Laptop" (the one added a moment before) and removes it from the shop
          */
+        shop.removeProductByName("Laptop");
     }
 
     private void changePriceForOneProduct() {
         /**
          * manager searches for a product with the name "Shirt" (the one added a moment before) and changes its price to 20
          */
+        shop.changePriceForProduct("Shirt", 20);
     }
 
     private void removeAllProductsWithCategory(Category category) {
         /**
          * manager removes all products with the category FOOD
          */
+        shop.removeProductByCategory(Category.FOOD);
```

Comment:
probably would be best to have here the parameter from the method because now this method always will remove FOOD regardless of the supplied value to the method